.github/workflows/Build.yml

```diff
@@ -75,19 +75,24 @@ jobs:
           TIMESTAMP="$(date -u +%F-%H%Z)"
           export TIMESTAMP
           echo "TIMESTAMP=${TIMESTAMP}" >> "$GITHUB_OUTPUT"
-      - name: Get all changed package files
-        id: changed-ruby-files
+      - name: Get changed files
+        id: changed-files
         uses: tj-actions/changed-files@v46
         with:
           base_sha: master
-          files: packages/*.rb
-          since_last_remote_commit: true
+          files_yaml: |
+            packages:
+              - packages/*.rb
       - name: Export variables to github context
         id: changed-packages
         run: |
-            # Convert "packages/foo.rb packages/bar.rb" (from steps.changed-ruby-files.outputs.all_changed_files) into "foo bar"
-            echo "CHANGED_PACKAGES=$(echo "${{ steps.changed-ruby-files.outputs.all_changed_files }}" | xargs basename -s .rb | xargs)" >> "$GITHUB_ENV"
-            echo "CHANGED_PACKAGES=$(echo "${{ steps.changed-ruby-files.outputs.all_changed_files }}" | xargs basename -s .rb | xargs)" >> "$GITHUB_OUTPUT"
+            if [[ -z "${{ steps.changed-files.outputs.packages_all_changed_files }}" ]]; then
+              echo "Branch ${{ env.BRANCH }} has no changed package files."
+              exit 1
+            fi
+            # Convert "packages/foo.rb packages/bar.rb" (from steps.changed-files.outputs.packages_all_changed_files) into "foo bar"
+            echo "CHANGED_PACKAGES=$(echo "${{ steps.changed-files.outputs.packages_all_changed_files }}" | xargs basename -s .rb | xargs)" >> "$GITHUB_ENV"
+            echo "CHANGED_PACKAGES=$(echo "${{ steps.changed-files.outputs.packages_all_changed_files }}" | xargs basename -s .rb | xargs)" >> "$GITHUB_OUTPUT"
```

Comment:
Why do we need 2 environment variables that hold the same values?

Reply:
`GITHUB_ENV` allows other _steps_ in the job to use the variable.
`GITHUB_OUTPUT` allows other _jobs_ in the workflow to use the variable.

Reply:
Why not just reference one variable and keep it simple?